roles/metrics/README.md

```diff
@@ -0,0 +1,25 @@
+theforeman.operations.metrics
```

Comment:
should this be called `metrics` or `monitoring`? the guide is called monitoring, but it's really more like "metrics"

Reply:
The role itself feels like it's about metrics. And metrics can be combined into monitoring. So yes, I think you keep this name.